packages/components/select/src/select.stories.ts

```diff
@@ -78,24 +146,68 @@ export const Basic: StoryObj = {
         <sl-select-option>🦊 Fox</sl-select-option>
       </sl-select-option-group>
       <sl-select-option>🤖 Robot</sl-select-option>
-    </sl-select>
-    <sl-button>To focus</sl-button>
-  `
+    `
+  }
+};
+
+export const OptionOverflow: Story = {
+  args: {
+    hint: 'This field has a lot of options, try scrolling through them.',
+    options: html`
+      ${Array.from(
+        { length: 100 },
+        (_, i) => html`<sl-select-option value="${i + 1}">Option ${i + 1}</sl-select-option>`
+      )}
+    `
+  }
+};
+
+export const Required: Story = {
+  args: {
+    hint: 'This field is required, if you leave it empty you will see an error message when clicking the button.',
+    required: true
+  }
+};
+
+export const TextOverflow: Story = {
+  args: {
+    placeholder:
+      'Cupidatat adipisicing adipisicing dolore in ea ea magna culpa Lorem aute veniam in. Laboris ea pariatur velit adipisicing pariatur aliqua Lorem est aliqua Lorem minim excepteur.',
+    options: html`
+      <sl-select-option value="1">
+        Voluptate sint ullamco proident cillum sint nostrud laborum labore et ad minim veniam eiusmod.
+      </sl-select-option>
+      <sl-select-option value="2">Consequat cupidatat amet sunt laborum laborum quis.</sl-select-option>
+      <sl-select-option value="3">
+        Culpa cillum nulla aute non quis deserunt minim sit magna. Consectetur in laborum mollit ea cillum dolor est ut
+        deserunt qui nostrud deserunt. Labore adipisicing anim non sint.
+      </sl-select-option>
+    `
+  }
+};
+
+export const Valid: Story = {
+  args: {
+    hint: 'After clicking the button, this field will show it is valid.'
+  }
 };
 
-const options = html`
-  <sl-select-option>🐷 Pig</sl-select-option>
-  <sl-select-option selected>🐨 Koala</sl-select-option>
-  <sl-select-option>🐼 Panda</sl-select-option>
-  <sl-select-option>🦊 Fox</sl-select-option>
-`;
 export const All: StoryObj = {
```

Comment:
Can you please update `All` story to show `valid` and `invalid` state as well (with the right styling)? :)

<img width="1522" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/0daedbb7-9212-46b7-ab13-87a1ec4abd7e">